deployment/united-manufacturing-hub/values.yaml

```diff
@@ -934,7 +942,22 @@ grafana:
           # <bool> allow users to edit datasources from the UI.
           isDefault: true
           editable: false
-        # <string, required> name of the datasource. Required
+        # https://grafana.com/docs/grafana/latest/datasources/postgres/
+        - name: Postgres 
+          type: postgres
+          url: united-manufacturing-hub:5432
+          user: $GRAFANAREADER_USER
+          secureJsonData:
+            password: $GRAFANAREADER_PASSWORD
+          jsonData:
+            database: factoryinsight
+            sslmode: 'disable' # disable/require/verify-ca/verify-full
+            maxOpenConns: 100 # Grafana v5.4+
+            maxIdleConns: 100 # Grafana v5.4+
+            maxIdleConnsAuto: true # Grafana v9.5.1+
+            connMaxLifetime: 14400 # Grafana v5.4+
+            postgresVersion: 1308 # 903=9.3, 904=9.4, 905=9.5, 906=9.6, 1000=10
```

Comment:
can you double check with the postgres version that we are using? we should be at 14

Reply:
We are using 13.8